ui/src/router/components/form/components/pyfunc_config/PyFuncDeploymentPanel.js

```diff
@@ -0,0 +1,83 @@
+import React, { useContext } from "react";
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormRow,
+  EuiSpacer,
+  EuiSuperSelect,
+} from "@elastic/eui";
+import { Panel } from "../Panel";
+import { EuiFieldDuration } from "../../../../../components/form/field_duration/EuiFieldDuration";
+import { useOnChangeHandler } from "../../../../../components/form/hooks/useOnChangeHandler";
+import { FormLabelWithToolTip } from "../../../../../components/form/label_with_tooltip/FormLabelWithToolTip";
+import EnsemblersContext from "../../../../../providers/ensemblers/context";
+
+export const PyFuncDeploymentPanel = ({
+  values: { ensembler_id, timeout },
+  onChangeHandler,
+  errors = {},
+}) => {
+  const { ensemblers } = useContext(EnsemblersContext);
+
+  let options = Object.values(ensemblers).reduce((pyfunc_ensemblers, val) => {
+    pyfunc_ensemblers.push({
+      value: val["id"],
+      inputDisplay: val["name"],
+    });
+    return pyfunc_ensemblers;
+  }, []);
+
+  const { onChange } = useOnChangeHandler(onChangeHandler);
+  const selectedOption = options.find(
+    (option) => option.value === ensembler_id
+  );
+
+  return (
+    <Panel title="Deployment">
+      <EuiForm>
+        <EuiFlexGroup direction="row">
+          <EuiFlexItem grow={2}>
+            <EuiFormRow
+              fullWidth
+              label={
+                <FormLabelWithToolTip
+                  label="Pyfunc Ensembler *"
+                  content="Select the Pyfunc Ensembler to be used in your deployment"
+                />
+              }
+              // isInvalid={!!errors.pyfunc_config}
+              // error={errors}
```

Comment:
there should be error handling where Ensembler Type is set to pyfunc, but pyfunc ensembler is not selected. User shouldn't be able to navigate further -as per enricher/ensembler docker image type selected but docker image not populated

Reply:
Oops those were left out unintentionally, I've got those error handling and schema validation covered now with the new changes.